test/lexers/vue-lexer.test.js

```diff
@@ -0,0 +1,31 @@
+import { assert } from 'chai'
+import VueLexer from '../../src/lexers/vue-lexer'
+
+describe('VueLexer', () => {
+  it('extracts keys from template & js', (done) => {
+    const Lexer = new VueLexer()
+    const content = "<template><p>{{ $t('first') }}</p><template><script>export default { mounted() { this.$i18n.t('second'); } }</script>"
```

Comment:
The project code style require each loc to be max 100 char long. 

cc @vincaslt a prettifier would be really nice I believe.